target/bin/fail2ban

```diff
@@ -3,7 +3,11 @@
 # shellcheck source=../scripts/helpers/index.sh
 source /usr/local/bin/helpers/index.sh
 
-function __usage { echo "Usage: ${0} [<unban> <ip-address>]" ; }
+function __usage {
```

Comment:
This `__usage` definition does not really look like what, lets say `addmailuser` looks like. I think `addmailuser` looks nicer. Having an `echo` on each new line seems kind of awkward to me.

Reply:
Personally I think, more echo's are looking much more better, than one echo/printf across multiple lines. But no need to discuss this further 😆 

I don't think we need a full blown `__usage` function like in addmailuser. However, you are right, to streamline this I would go for:

`function __usage { echo "Usage: ${0} [<ban|unban> <ip-address>]" ; }`

Beside the question of using a simple/full-blown `__usage` function, I just noticed this fact: `__usage` is never called 😉 If no argument is supplied, the script will list all banned IPs. In comparison, other scripts are calling `__usage`, if no parameter is supplied.

Some options to choose from:

1. Remove `__usage` function. But then this script is not 100% like the others.
2. Keep `__usage` function with dummy code: `function __usage() { :; }`
3. Change the script, so that listing banned IPs needs an explicit argument like `show` or `list` --> `./setup.sh fail2ban list`. Then usage could be displayed, if no parameter is given. This would be a breaking change.
4. `__usage` will only be called, when `help` parameter is given.

I tend to option 4 at the moment or maybe 3.

BTW: I think the options ban/unban are so self-explanatory, that I cannot imagine, someone will every try to see the usage explanation.

Reply:
> I tend to option 4 at the moment or maybe 3.

I'd go for option 4 as well.

> BTW: I think the options ban/unban are so self-explanatory, that I cannot imagine, someone will every need the usage explanation.

Me too 👍🏼